addons/account_accountant/migrations/9.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='utf-8'?>
+<openerp>
+  <data>
+    <record id="module_install_notification" model="mail.message">
```

Comment:
This one is not needed as this a trick for Odoo for notifying all employees a module installation, and I'm afraid this would be triggered also on the migration.